Patient Registry System/PatientRegistrySystem.DB/Repos/GenericRepository/GenericRepository.cs

```diff
@@ -0,0 +1,387 @@
+﻿using PatientRegistrySystem.DB.Models.DbModels;
+using PatientRegistrySystem.DB.Repos.AplicationUserRepository;
+using PatientRegistrySystem.DB.Repos.DbGenericRepository;
+using PatientRegistrySystem.Domain.Dto;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace PatientRegistrySystem.DB.Repos.GenericRepository
+{
+    public class GenericRepository<DbEntity, DomainEntity1, DomainEntity2> : IGenericRepository<DbEntity, DomainEntity1, DomainEntity2>
```

Comment:
i this only related to the application users? not sure why it's called GenericRepository and what's the purpose of this repo.

I thought DbGeneric would be enough (i prefer calling this one GenericRepository), and use it in the actual repositories like UserRepository : DbGeneric<...>...